corehq/apps/reports/standard/deployments.py

```diff
@@ -308,6 +319,13 @@ def process_rows(self, users, fmt_for_export=False):
                         commcare_version = _get_commcare_version(device['commcare_version'])
             if last_sub and last_sub.get('submission_date'):
                 last_seen = string_to_utc_datetime(last_sub['submission_date'])
+            if last_sub:
+                last_build_profile_id = last_sub.get('build_profile_id')
+                if last_build_profile_id:
+                    last_build_profile_name = _("Unknown")
+                    build_profiles = self._get_app_details(last_sub['app_id'])['build_profiles']
```

Comment:
this could produce a `KeyError`

Reply:
oh yes, was wondering why you suggested the `get`, thanks. updated.